### DISCUSSION TITLE
resolvergen: package not found for importPath

### DISCUSSION THREAD

**Discussion by alexisvisco** [99designs/gqlgen#1796] [Q&A]
### What happened?

```
❯ go run github.com/99designs/gqlgen init
validation failed: packages.Load: err: exit status 1: stderr: go: gitlab.com/miimosa/lab/gateway-graphql-to-any/graph/generated: package github.com/vektah/gqlparser/v2 imported from implicitly required module; to add missing requirements, run:
        go get github.com/vektah/gqlparser/v2@v2.1.0
go: gitlab.com/miimosa/lab/gateway-graphql-to-any/graph/generated: package github.com/vektah/gqlparser/v2/ast imported from implicitly required module; to add missing requirements, run:
        go get github.com/vektah/gqlparser/v2/ast@v2.1.0

err: exit status 1: stderr: go: gitlab.com/miimosa/lab/gateway-graphql-to-any/graph/generated: package github.com/vektah/gqlparser/v2 imported from implicitly required module; to add missing requirements, run:
        go get github.com/vektah/gqlparser/v2@v2.1.0
go: gitlab.com/miimosa/lab/gateway-graphql-to-any/graph/generated: package github.com/vektah/gqlparser/v2/ast imported from implicitly required module; to add missing requirements, run:
        go get github.com/vektah/gqlparser/v2/ast@v2.1.0


Exec "go run ./server.go" to start GraphQL server


❯ go run github.com/99designs/gqlgen generate
resolvergen: package not found for importPath: gitlab.com/miimosa/lab/gateway-graphql-to-any/graphexit status 1
```

### What did you expect?

I just edited something in the generated schema (type change from string to int)  and I wanted to generate the schema again but I got this error with importPath. 

By the way my go.mod file has `module gitlab.com/miimosa/lab/gateway-graphql-to-any` the good path and there is the package graph inside the repository because the init command created it. 

And the verbose flag doesn't help there is no additional information. 

### Minimal graphql.schema and models to reproduce

The default schema but I changed just a type and I don't think this is the problem. 

### versions
 - `gqlgen version`? v0.13.0
 - `go version`? go1.16.4 linux/amd64
 - dep or go modules? go module

**Comment by giorgenes** [comment]
I'm experiencing the same on macos with go 1.16.5.

Just followed the "getting started" tutorial and I can't generate the models

**Comment by chrislentz** [comment]
I believe I was able to find a work around for this issue. I was originally receiving this same issue. I manually deleted the graph/schema.resolvers.go file. Once I did this, I received a new error asking to manually install 2 new packages:

```
package github.com/vektah/gqlparser/v2 imported from implicitly required module; to add missing requirements, run: go get github.com/vektah/gqlparser/v2@v2.1.0

package github.com/vektah/gqlparser/v2/ast imported from implicitly required module; to add missing requirements, run: go get github.com/vektah/gqlparser/v2/ast@v2.1.0
```

After running those 2 `go get` commands, `gqlgen generate` was able to completed successfully.
